lib/rock_rms/resources/person.rb

```diff
@@ -53,6 +53,22 @@ def create_person(first_name:, last_name:, email:, connection_status_value_id: n
         post(people_path, options)
       end
 
+      def create_business(last_name:, email:, connection_status_value_id: nil, record_status_value_id: nil, record_type_value_id: 2)
+        options = {
+          'IsSystem' => false,
+          'LastName' => last_name,
+          'Email' => email,
+          'Gender' => 1,
```

Comment:
What gender is `1`? I think it gets set by default and may not be needed.

Reply:
I don't know. I copied it over from create_person. I don't know enough to know it's safe to remove, but I have no objection if you remove it.